### PR TITLE
svg styles for playlist subscription button

### LINKED ISSUE #14: playlist subscribe button not themed right in dark mode
screenshot_2016-03-08_22-10-56

screenshot_2016-03-08_22-11-00

It appears when you open a playlist created by someone else.

### PR DESCRIPTION
**Pull request by chrismou**
fixes #14

Here's a couple of screenshots of the fixed styles from Radiant.  I've ported the styles over as is.

screenshot 2016-03-11 11 31 58

screenshot 2016-03-11 11 32 03

### CODE REVIEW

File: lib/style.css [hunk 1]
```diff
@@ -461,3 +461,29 @@ button.goog-buttonset-default:hover {
...
+paper-button#unsubscribe-playlist-button:hover .playlist-unsubscribe {
+    color: <<FORE_PRIMARY>>;
+}
+
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:unsubscribe"] path:last-child
+{
+    fill: <<FORE_PRIMARY>>;
+}
+
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:subscribed"] path
```
Comment (MarshallOfSound) [inline lib/style.css]: Can we make this comma separated :smile: 

so like

``` css
paper-button#unsubscribe-playlist-button iron-icon[icon="sj:unsubscribe"] path:last-child,
paper-button#unsubscribe-playlist-button iron-icon[icon="sj:subscribed"] path {
    fill: <<FORE_PRIMARY>>;
}
```

Comment (chrismou) [inline lib/style.css]: Ah yeah, good spot. I tackled them both separately because they both have different numbers of child elements. Woops. 2 secs.

**Comment by chrismou** [comment]
Sorted

File: lib/style.css [hunk 2]
```diff
@@ -461,3 +461,25 @@ button.goog-buttonset-default:hover {
...
 }
+
+paper-button#unsubscribe-playlist-button .playlist-subscribed,
+paper-button#unsubscribe-playlist-button:hover .playlist-unsubscribe {
+    color: <<FORE_PRIMARY>>;
+}
+
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:unsubscribe"] path:last-child,
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:subscribed"] path
+{
```
Comment (twolfson) [inline lib/style.css]: We should prob keep the opening brace on the same line as the last selector for consistency in our CSS

**Comment by chrismou** [comment]
Fixed braces

**Comment by chrismou** [comment]
Anyone spot any other issues or am I good to land it?

**Comment by MarshallOfSound** [comment]
Sorry this one must have slipped through my notifications.

LGTM :+1:

**Pull request merged**